.travis/cc-report.sh

```diff
@@ -5,8 +5,8 @@ if [ ! -f $(dirname $0)/cc-test-reporter ]; then
   exit 1;
 fi
 
-if [ "$TRAVIS_PULL_REQUEST" == "false" -a "$TRAVIS_BRANCH" != "master" ]; then
-  echo "Skipping coverage uploads for push builds on $TRAVIS_BRANCH"
+if [ "$TRAVIS_BRANCH" == "master" ]; then
+  echo "Skipping coverage uploads for push builds on master"
```

Comment:
We want coverage to be reported on master, that's what the main coverage report on codeclimate is based on. I think on reflection we just remove this condition entirely.